### PR TITLE
add tags endpoints

### PR DESCRIPTION
**Pull request by fschade**
add graph endpoints for tags
fixes https://github.com/owncloud/libre-graph-api/issues/59

### CODE REVIEW

**Review by micbar — APPROVED** [review]
Please add two examples for the put and delete requests

**Review by micbar — CHANGES_REQUESTED** [review]
There is a way to mark body properties required.

**Comment by fschade** [comment]
@micbar can you re-check if the examples are correct?

**Comment by micbar** [comment]
https://github.com/owncloud/libre-graph-api/blob/main/api/openapi-spec/v1.0.yaml#L1606-L1608

File: api/openapi-spec/v1.0.yaml [hunk 1]
```diff
@@ -1565,6 +1565,57 @@ paths:
           description: Success
         default:
           $ref: '#/components/responses/error'
+  '/extensions/org.libregraph/tags':
```
Comment (kulmann) [inline api/openapi-spec/v1.0.yaml:1568]: we can't register the domain `libregraph.org` (already taken). I'd recommend registering e.g. `libregraph.io` and use the namespace accordingly.

Comment (fschade) [reply api/openapi-spec/v1.0.yaml:1568]: done

Comment (fschade) [reply api/openapi-spec/v1.0.yaml:1568]: reverted, back to org.libregraph

**Review by micbar — APPROVED** [review]

**Pull request merged**